pkg/registry/core/service/strategy.go

```diff
@@ -179,6 +179,16 @@ func dropServiceDisabledFields(newSvc *api.Service, oldSvc *api.Service) {
 	if !utilfeature.DefaultFeatureGate.Enabled(features.ServiceTopology) && !topologyKeysInUse(oldSvc) {
 		newSvc.Spec.TopologyKeys = nil
 	}
+
+	// Clear AllocateLoadBalancerNodePorts if ServiceLBNodePortControl is not enabled
```

Comment:
Why are there 2 blocks doing the same thing here?

Reply:
Ooops, I did not see that it was already there in some form :blush: Removing the second.